### PR TITLE
DAP-9 - Make Backend server URL configurable

### PR DESCRIPTION
**Pull request by sainingo**
# Worked on backend server URL configurable

Issue I worked on

see https://ampath.atlassian.net/browse/DAP-9

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: config/BASE_URL.ts
Comment (kipchubett) [inline config/BASE_URL.ts:1]: Rename to `Constant.ts`

File: src/components/authentication/Login.tsx [hunk 1]
```diff
@@ -1,68 +1,152 @@
...
+        <p data-testid="state-error" className="text-red-500 font-bold">
+          {errorMessage}
+        </p>
+      </div>
+    );
+  };
+
+  useEffect(() => {
+  
+    localStorage.setItem("selectedServerUrl", selectedServer);
```
Comment (kipchubett) [inline src/components/authentication/Login.tsx]: Extract `selectedServerUrl` into a constant

File: src/components/patientSearch/Patient.tsx [hunk 2]
```diff
@@ -34,15 +32,15 @@ const PatientSearch = () => {
...
-        localStorage.removeItem("Banner")
+        // const banner = localStorage.getItem("Banner")
+        //   if(banner!==null){
+        //     swal({
+        //       title:'Success!',
+        //       text:"Logged In",
+        //       icon:"success",
+        //   })
+        //   }
+        // localStorage.removeItem("Banner")
```
Comment (kipchubett) [inline src/components/patientSearch/Patient.tsx]: Clean up this

**Review by kipchubett — CHANGES_REQUESTED** [review]

File: config/BASE_URL.ts
Comment (sainingo) [reply config/BASE_URL.ts:1]: :+1:

File: src/components/authentication/Login.tsx (hunk 1 above)
Comment (sainingo) [reply src/components/authentication/Login.tsx]: :+1:

File: src/components/patientSearch/Patient.tsx (hunk 2 above)
Comment (sainingo) [reply src/components/patientSearch/Patient.tsx]: :+1:

**Pull request merged**